blocks/init/src/Blocks/components/paragraph/components/paragraph-editor.js

```diff
@@ -31,15 +39,152 @@ export const ParagraphEditor = (attributes) => {
 		selector(blockClass, blockClass, selectorClass),
 	]);
 
+	/**
+	 * Returns string parsed as nodes, ready for inserting into RichText.
+	 *
+	 * @param {string} inputText HTML string for parsing.
+	 */
+	const processTag = (inputText) => {
+		// If it's only text wrap it in span for processing
+		if (!inputText.startsWith('<')) {
+			inputText = `<span>${inputText}</span>`;
+		}
+
+		let output = [];
+
+		const matches = [...inputText.matchAll(/<.+?>.+?<\/.+?>/g)];
+
+		for (const [match] of matches) {
+			const elementTag = match.match(/<.+?>/g)[0];
+			const filteredTagName = elementTag.replaceAll(/[<>/\s]+/g, '');
+			const innerText = match.replaceAll(/<.+?>/g, '');
+
+			if (filteredTagName === 'span') {
+				output = [...output, document.createTextNode(`${innerText}`)];
+			} else {
+				let newElement = document.createElement(filteredTagName);
+				newElement.appendChild(document.createTextNode(innerText));
+
+				output = [...output, newElement];
+			}
+		}
+
+		return output;
+	}
+
+	/**
+	 * Process content and paste in into blocks smartly.
+	 *
+	 * @param {string} textFromClipboard Text retrieved from clipboard.
+	 */
+	const handlePaste = async (textFromClipboard) => {
```

Comment:
You've tagged this as `async` but I don't see `await` anywhere. What am I missing?

Reply:
You're not missing anything, I needed it at one point, but should've checked at the end if I still need it 😅